jetstream/plugins/machine_validation.py

```diff
@@ -0,0 +1,35 @@
+from atmosphere.plugins.machine_validation import MachineValidationPlugin
+from django.conf import settings
+from threepio import logger
+
+
+class JetstreamValidation(MachineValidationPlugin):
+    """
+    Represents the current strategy being used by Jetstream
+
+    Default metadata_key is: `atmosphere_catalog`
```

Comment:
So `atmosphere_catalog="false"` would still result in the image being added to the catalog? How about we also require the metadata value to be truthy?

Reply:
Two of the things I wanted to discuss during the review:

1) What should we use as the default metadata_key
2) Do we want to treat metadata as boolean values, or simply require the existence of the key itself.

I'm fine either way..

Reply:
Follow-up after taking suggestions:
1. For "default-whitelist": `atmo_image_exclude`  and for "default-blacklist": `atmo_image_include`
2. Truthy! support `true/false` and `yes/no`

Reply:
sounds good @steve-gregory!